subsys/usb/class/audio/usb_audio_internal.h

```diff
@@ -511,7 +548,7 @@ struct dev##_descriptor_##i {						\
 	.bLength = sizeof(struct cs_as_ad_ep_descriptor),	\
 	.bDescriptorType = USB_CS_ENDPOINT_DESC,		\
 	.bDescriptorSubtype = 0x01,				\
-	.bmAttributes = 0x00,					\
+	.bmAttributes = 0x01,					\
```

Comment:
Shouldn't this vary depending on sizeof frequencies table?

Reply:
It can, or it could be a feature to enable. I'm thinking that if there is only one frequency available it can still report that it is possible to set and get that one frequency.